.github/workflows/linter.yml

```diff
@@ -27,6 +27,6 @@ jobs:
           DEFAULT_BRANCH: master
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VALIDATE_MD: true
-          MARKDOWN_CONFIG_FILE: .markdownlint.yml
+          MARKDOWN_CONFIG_FILE: .github/markdownlint.yml
```

Comment:
nit: if we move `markdownlint.yml` into a `linter` folder in `.github/` we can remove this line.

Reply:
Ooh, even better!